tests/testthat/test-reporter.R

```diff
@@ -1,3 +1,12 @@
+
+# Disable srcrefs because they differ across systems
+withr::local_options(list(rlang_trace_format_srcrefs = FALSE))
+
+skip_if(getRversion() < "3.4", "Fails because of new `eval()` call structure")
```

Comment:
Does this mean testthat now depends on R 3.4+? What happens if this code is run on R 3.2 or 3.3?

Reply:
Nothing bad happens, the output is just different. I would prefer to still run the code to detect unexpected errors, see comment in https://github.com/r-lib/testthat/pull/921#issuecomment-519494083.

Reply:
(oops the comment is about `skip_on_cran()`, not the skip on earlier versions)